packages/documentation/src/pages/forms/creating-a-form-config-file.md

```diff
@@ -22,8 +21,138 @@ The form config itself is an object with many properties that determine how your
 
 The `schema` and `uiSchema` contain the essential information to build all of the form fields. For more information, see "[About the `schema` and `uiSchema` objects](/forms/about-the-schema-and-uischema-objects)." For a comprehensive example of a form config, with descriptions of the main properties you might include, see "[config options](/forms/config-options)".
 
-### How React components are rendered
+## How React components are rendered
 
-The VAFS code uses the form config to determine which React components to render in order to build your form, usually automatically. React components require props to render properly. These are passed down through the top-level `Form` component, which takes the entire form config itself as a prop. The `Form` component is also connected to the store, so it has access to the form data as well.
+The VA Forms Library code uses the form config to determine which React components to render in order to build your form, usually automatically. React components require props to render properly. These are passed down through the top-level `Form` component, which takes the entire form config itself as a prop. The `Form` component is also connected to the store, so it has access to the form data as well.
 
 For more information, see "[About the component hierarchy](/forms/component-hierarchy)."
+
+## How to structure your form
+
+When creating forms using the VA Forms Library it can be a little confusing figuring out how to structure your code so that it will be easy to maintain and easy to understand later. 
+
+All forms begin with a `form.js` file inside of a `/config` folder. This file is generated by the form generator and is intended to house your `schema` and your `uiSchema`. This structure works well for simple forms, but for more advanced forms we recommend that you keep this file but also break your `schema` and your `uiSchema` across smaller files that are easier to maintain. Here is a rough example of what your `form.js` will look like when you have your `schema` and `uiSchema` inside it:
```

Comment:
Has this been edited? Looks like mentions of "we" were removed from the existing documentation, but not the new stuff.